src/Tribe/Main.php

```diff
@@ -1412,9 +1428,9 @@ public function displayEventOrganizerDropdown( $postId ) {
 			?>
 			<tr class="">
 				<td style="width:170px">
-					<label for="saved_organizer"><?php printf( __( 'Use Saved %s:', 'tribe-events-calendar' ), $this->singular_organizer_label ); ?></label>
+					<label for="saved_organizer"><?php echo esc_html( sprintf( __( 'Use Saved %s:', 'tribe-events-calendar' ), $this->singular_organizer_label ) ); ?></label>
```

Comment:
this escaping could cause a bug.  (I know you are confirming now based on an offline conversation)